telegraf/manifests/deis-monitor-telegraf-daemon.yaml

```diff
@@ -41,6 +41,13 @@ spec:
             value: "metrics"
           - name: "NSQ_ENDPOINTS"
             value: http://$(DEIS_NSQD_SERVICE_HOST):$(DEIS_NSQD_SERVICE_PORT_HTTP)
+          - name: DEIS_LOGGER_REDIS_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                name: logger-redis-creds
+                key: password
+          - name: "REDIS_SERVERS"
+            value: tcp://:$(DEIS_LOGGER_REDIS_PASSWORD)@$(DEIS_LOGGER_REDIS_SERVICE_HOST):$(DEIS_LOGGER_REDIS_SERVICE_PORT)
```

Comment:
Well this is what @krancour did in the chart pr that is adding redis - https://github.com/deis/charts/pull/293

I'm not sure I 100% agree with it but I'm also not sure of another way to do it.